website/src/components/highlighter.tsx

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { usePrismTheme } from "@docusaurus/theme-common";
+import Highlight, { defaultProps } from "prism-react-renderer";
+
+interface HighlighterProps {
+  children: string;
+  tokenReplacement?: (token: PrismToken) => PrismToken;
+}
+
+export type PrismToken = {
+  types: string[];
+  content: string;
+  empty?: boolean;
+};
+
+/**
+ * A highlighter with built in overrides for Functionless.
+ *
+ * Function - Generally classified as a built in, this highlighter removes the built-in
+ * to support Functionless {@link Function}.
+ */
+export function FunctionlessHighlighter({
+  children,
+  tokenReplacement,
+}: HighlighterProps) {
+  return (
+    <Highlighter
+      tokenReplacement={(token) => {
```

Comment:
Nice one. I am impressed you actually solved it haha.

Reply:
Everything is an AST. All I see anymore are ASTs.